code/modules/shuttle/marine_dropship.dm

```diff
@@ -303,6 +303,8 @@
 #define ALIVE_HUMANS_FOR_CALLDOWN 0.1
 
 /datum/game_mode/proc/can_summon_dropship(mob/user)
+	if(!(flags_round_type & MODE_HIJACK_POSSIBLE))
+		return
```

Comment:
that's an additionel check that didn't exist.

Useless in this PR, useless overall but is usefull in case of future mode. 

Tell me if you want it to be deleted, since it's not in this PR scope, but without it MODE_HIJACK_POSSIBLE is not used